crates/fuel-core/src/database/sealed_block.rs

```diff
@@ -51,6 +54,23 @@ impl OnChainIterableKeyValueView {
         }
     }
 
+    /// Returns a `Block`, which is sealed, by height
```

Comment:
I find this docstring a bit confusing as we don't return a `SealedBlock` in this function. We chatted offline about this, and the important thing here is that the block has been sealed (i.e. it is signed) but downstream we only need the actual block.

```suggestion
    /// Returns a `Block` by it's height. Only returns a block if it has been sealed.
```

Reply:
New information based on @xgreenx's comment below and an offline discussion. The block importer will only persist blocks that are sealed so we can omit this from the docstring altogether
```suggestion
    /// Returns a `Block` by its height.
```